rc/lsp.kak

```diff
@@ -1873,6 +1906,31 @@ define-command lsp-stop-on-exit-disable -docstring "Don't end kak-lsp session on
     alias global lsp-exit lsp-exit-editor-session
 }
 
+define-command lsp-inlay-hints-enable -params 1 -docstring "lsp-inlay-hints-enable <scope>: Enable inlay hints for <scope>" %{
+    add-highlighter "%arg{1}/lsp_inlay_hints" replace-ranges lsp_inlay_hints
+    hook -group lsp-inlay-hints %arg{1} BufReload .* lsp-inlay-hints
```

Comment:
I guess this means we should make `lsp-inlay-hints` hidden because most users will be happy with `lsp-inlay-hints-enable`